datadog_checks_dev/datadog_checks/dev/tooling/commands/test.py

```diff
@@ -276,7 +277,12 @@ def test(
                 if not cov_keep:
                     echo_info('\n---------- Coverage report ----------\n')
 
-                    result = run_command('coverage report --rcfile=../.coveragerc')
+                    # Marketplace PRs should fail CI if coverage is not greater than 0
+                    if repo == 'marketplace':
+                        result = run_command('coverage report --rcfile=../.coveragerc --fail-under=1')
```

Comment:
Is there a way to specify `fail_under` in the coveragerc directly? It'd be nice to avoid hardcoding this value in ddev if possible 🙂

Reply:
Added new option to allow specifying `fail_under` value.

Reply:
Why did we add an option?

Reply:
See `fail_under` in https://coverage.readthedocs.io/en/latest/config.html#report

Reply:
Sounds good, I will use `fail_under` in `.coveragerc` file for marketplace instead.